lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/persistence/DBOAccessApproval.java

```diff
@@ -39,7 +37,6 @@
 	private long expiredOn;
 	private Long requirementId;
 	private Long accessorId;
-	private byte[] serializedEntity;
```

Comment:
leave this field and it's methods
remove translator's logic about previous migration.